conf/machine/include/imx-base.inc

```diff
@@ -63,6 +63,10 @@ UBOOT_BINARY ?= "u-boot.${UBOOT_SUFFIX}"
 IMX_DEFAULT_BOOTLOADER ??= "u-boot-fslc"
 UBOOT_SUFFIX ?= "${UBOOT_SUFFIX_pn-${IMX_DEFAULT_BOOTLOADER}}"
 
+# We need to export the original variable to allow it to be used when generating
+# wic based images.
+SPL_BINARY ?= "${SPL_BINARY_pn-${IMX_DEFAULT_BOOTLOADER}}"
+
```

Comment:
Was there a build error? Can you mention in the commit message how the problem manifested itself?

Reply:
We couldn't reference it on the wks file. It took me some time to find the root cause as the error wan't very useful.